tools/local/run-bfb-upload.sh

```diff
@@ -51,6 +51,34 @@ BFB_PARAMETERS=" \
     $EXTRA_PARAMS
 "
 
+if [ "$QC_NAME" == "qdrant-chaos-testing-three" ]; then
+  # shellcheck disable=SC2124
+  BFB_PARAMETERS=" \
+      --uri https://node-0-${QDRANT_CLUSTER_URL}:6334 \
+      --uri https://node-1-${QDRANT_CLUSTER_URL}:6334 \
```

Comment:
We don't pass `QDRANT_CLUSTER_URL` to this script. 

So you need to pass it [here](https://github.com/qdrant/haloperidol/blob/master/tools/run-bfb-upload.sh#L29) like other vars.